com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/handler/LaunchRequestHandler.java

```diff
@@ -58,7 +75,7 @@
     @Override
     public CompletableFuture<Response> handle(Command command, Arguments arguments, Response response, IDebugAdapterContext context) {
         LaunchArguments launchArguments = (LaunchArguments) arguments;
-        activeLaunchHandler = launchArguments.noDebug ? new LaunchWithoutDebuggingDelegate() : new LaunchWithDebuggingDelegate();
+        activeLaunchHandler = launchArguments.noDebug ? new LaunchWithoutDebuggingDelegate(this) : new LaunchWithDebuggingDelegate();
```

Comment:
don't reference 'this' in LaunchWithoutDebuggingDelegate, but reference a callback function